### ISSUE TITLE
Clearer test results with diffs

### ISSUE THREAD

**Issue by BurnzZ** [scrapinghub/scrapy-autounit#50]
At the moment, the test results doesn't have much details on which data points have changed during the test. It goes something like:

```
autounit/tests/my_spider/parse_book/test_fixture1.py:13:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
env2.7/lib/python2.7/site-packages/scrapy_autounit/utils.py:325: in test                                                                           self.assertEqual(fx_obj, cb_obj, 'Not equal!')
E   AssertionError: Not equal!
```

As discussed with @fcanobrash, this was due to some of the **time** related data points which was easily solved by having something like this in the settings: `'AUTOUNIT_SKIPPED_FIELDS': ['timestamp']`

If the test results shows the actual diff between fields, that would be awesome as it's much clearer. 💯

**Comment by andrewbaxter** [comment]
Maybe related to #19 ?  @fcanobrash I also got confused with a recent issue :) - I'll take this/#19 if that's alright.

**Comment by andrewbaxter** [comment]
Ah, this is specifically a Python 2 thing - Python 3 shows a rough diff of the values.  The fix for #19 should solve this anyway though.

**Comment by fcanobrash** [comment]
Solved in https://github.com/scrapinghub/scrapy-autounit/pull/51
